capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -67,13 +69,38 @@ export class SearchUserModal extends Component {
     this.setState({
       searchTerm: event.target.value,
       typingTimeout: setTimeout(async () => {
-        await this.getBooks(this.state.searchTerm)
+        await this.getUsers(this.state.searchTerm)
       }, 500)
     })
   }
 
   componentDidMount() {
     this.fetchCollaborators();
+
+    // TODO: Update ClubPage based on user input
+
+    if (this.props.club && this.props.type === 'clubs') {
+      if (this.props.club.memberIDs) {
+        if (this.props.club.memberIDs.length > 0) {
```

Comment:
Rather than having three nested conditions here inside of which you call `fetchMembers`, could you just `&&` together all the conditions in a single `if` statement?

Reply:
Ok